packages/vats/src/core/utils.js

```diff
@@ -63,6 +66,52 @@ export const feeIssuerConfig = {
   displayInfo: { decimalPlaces: 6, assetKind: AssetKind.NAT },
 };
 
+/**
+ * @template {AssetKind} K
+ *
+ * Create a brand that cannot ever produce mints, issuers, payments or purses.
+ *
+ * The allegedName becomes part of the brand in asset descriptions. The
+ * allegedName doesn't have to be a string, but it will only be used for
+ * its value. The allegedName is useful for debugging and double-checking
+ * assumptions, but should not be trusted.
+ *
+ * The assetKind will be used to import a specific mathHelpers
+ * from the mathHelpers library. For example, natMathHelpers, the
+ * default, is used for basic fungible tokens.
+ *
+ *  `displayInfo` gives information to the UI on how to display the amount.
+ *
+ * @param {string} allegedName
+ * @param {K} [assetKind=AssetKind.NAT]
+ * @param {AdditionalDisplayInfo} [displayInfo={}]
+ * @returns {Brand<K>}
+ */
+export const makeInertBrand = (
+  allegedName,
+  assetKind = /** @type {K} */ (AssetKind.NAT),
+  displayInfo = harden({}),
+) => {
+  const cleanDisplayInfo = coerceDisplayInfo(harden(displayInfo), assetKind);
+  return Far(`${allegedName} brand`, {
```

Comment:
is there some reason not to use `makeIssuerKit` and return only the brand?

Reply:
That's probably the best way.  Removing this one.